README.md

```diff
@@ -48,7 +52,13 @@ The interface for mutable usage is exactly the same except the source array is m
 ```javascript
 import set from 'arraysetjs'
 
-const arr = [1, 2, 3]
-set.mutable(arr).union([3, 4])        // a is now [1, 2, 3, 4]
+const a = [1, 2, 3]
+set.mutable(a).union([3, 4])          // a is now [1, 2, 3, 4]
+                                      // (which is also returned)
+
+// remove duplicates
+
+const a = [1, 2, 2, 3]
+set.mutable.fromArray(arr)            // a is now [1, 2, 3]
```

Comment:
I don't think an example is needed in the "Mutable usage" section because creation of a new arrayset instance is, by definition, an immutable action.  It's not harmful that `set.mutable` exposes `fromArray`, but it doesn't really make a lot of since since there's nothing to mutate.